src/lib/index.ts

```diff
@@ -22,29 +23,22 @@ export default class BundleChecker {
   private inputParams: IBundleCheckerParams;
 
   constructor(params: IBundleCheckerParams) {
-    this.inputParams = params; // TODO: perform default override of some params
+    this.inputParams = params;
     this.originalCwd = process.cwd();
   }
 
-  public async compareByFileExtension(): Promise<ITableRow[]> {
-    const { currentBranchReport, targetBranchReport } = await this.compareEachFile();
-
-    return getFormattedRows({
-      currentBranchReport: squashReportByFileExtension(currentBranchReport),
-      targetBranchReport: squashReportByFileExtension(targetBranchReport)
-    });
-  }
-
-  public async compare(): Promise<ITableRow[]> {
-    const { currentBranchReport, targetBranchReport } = await this.compareEachFile();
-    return getFormattedRows({ targetBranchReport, currentBranchReport }, this.workDir);
+  public async commentOnPr(result: IBundleCheckerReport) {
```

Comment:
why are we wrapping the `commentOnPr` function in here? can we omit this public method from the checker?